docs/NotebookInstructions.md

```diff
@@ -5,55 +5,38 @@ This example notebook demonstrates how to use the [Dataset Insights](https://git
 
 ## Workflow
 
-* If you do not have Conda installed on your computer, install Conda. We recommend [Miniconda](https://docs.conda.io/en/latest/miniconda.html).
+* Make sure [Docker Desktop](https://www.docker.com/products/docker-desktop) is installed.
 
-* Once Conda is installed: 
-  * On Mac OS, open a new terminal window.
-  * On Windows, you will need to open either Anaconda Prompt or Anaconda Powershell Prompt. These can be found in the Start menu.
+* Open a command line and use the following command to download and run the Dataset Insights Docker image, and mount your local folders to it:
 
-* Create a virtual environment named `synthdet_env` using Conda, and activate it:
-
-```bash
-conda create -n synthdet_env python=3.7
-conda activate synthdet_env
 ```
-
-* Install Dataset Insights:
-
-```bash
-pip install datasetinsights
+docker run -p 8888:8888 -v <datasets_path>:/data -v <synthdet_notebook_path>:/tmp -t unitytechnologies/datasetinsights:latest
 ```
 
-* Install Jupyter:
-
-```bash
-conda install jupyter
-```
+In the above command, `<datasets_path>` is the path to the folder where all locally generated SynthDet datasets are stored, and `<synthdet_notebook_path>` is the location at which the `SynthDet_Statistics.ipynb` notebook file is located inside your SynthDet repository (`repository_root/Notebooks`). 
```

Comment:
```suggestion
In the above command, `<datasets_path>` is the path to the folder containing SynthDet datasets are stored, and `<notebooks_path>` is the full path to the `Notebooks` directory in this repository. 
```
I found the wording a bit confusing - was it supposed to point to the notebook itself or the directory containing it? Tried to clarify.

Reply:
It is supposed to point to the Notebooks folder. Thanks, reads better now :)